app/models/host.rb

```diff
@@ -198,35 +198,33 @@ class Host < ApplicationRecord
   supports_not :vmotion_enabled
   supports     :ipmi do
     if ipmi_address.blank?
-      unsupported_reason_add(:ipmi, _("The Host is not configured for IPMI"))
+      _("The Host is not configured for IPMI")
     elsif authentication_type(:ipmi).nil?
-      unsupported_reason_add(:ipmi, _("The Host has no IPMI credentials"))
+      _("The Host has no IPMI credentials")
     elsif authentication_userid(:ipmi).blank? || authentication_password(:ipmi).blank?
-      unsupported_reason_add(:ipmi, _("The Host has invalid IPMI credentials"))
+      _("The Host has invalid IPMI credentials")
     end
   end
 
   supports :start do
-    valid_states = respond_to?(:vim_power_up_from_standby) ? %w[off standby] : "off"
+    valid_states = respond_to?(:vim_power_up_from_standby) ? %w[off standby] : %w[off]
```

Comment:
as a followup, will move this into vmware

For now, want to just focus on one repo at a time.

Reply:
https://github.com/ManageIQ/manageiq-providers-vmware/pull/809